test/utils/cryptography/ECDSA.test.js

```diff
@@ -61,6 +77,15 @@ contract('ECDSA', function (accounts) {
           await expectRevert(this.ecdsa.recover(TEST_MESSAGE, signature), 'ECDSA: invalid signature \'v\' value');
         });
       });
+
+      context('short 2098 format', function () {
+        it('works', async function () {
```

Comment:
A lot of the tests in this file have `it('works'` and `it('reverts'` as the only tests in a describe block. This is really unnecessary, we should be writing the test directly as in: `it('recovers EIP2098 format', ...` and `it('reverts with ...`.

Can you clean this up while we're working on this file?

Reply:
Lets discuss that in sync. You know I'm bad with names, and its not just about the new tests, its also about the old ones...

Reply:
I'm ok with changing to `works with <name of describe>` and `reverts with <name of describe>`. If you don't feel comfortable with it let's move ahead and merge if this is ready and we can talk about the test titles afterwards.